pkg/piecestore/psserver/psdb/psdb_test.go

```diff
@@ -274,6 +276,52 @@ func TestBandwidthUsage(t *testing.T) {
 	})
 }
 
+func TestMigration(t *testing.T) {
+	_, err := exec.LookPath("sqlite3")
+	if err != nil {
+		t.Skip("unable to find sqlite3 executable")
+	}
+
+	// find latest version
+	migration := psdb.Migration()
+	migration.Now = "now"
+
+	latestVersion := 0
+	// TODO support missing intermediate versions
+	for _, step := range migration.Steps {
+		if latestVersion < step.Version {
+			latestVersion = step.Version
+		}
+	}
+
+	for version := 0; version <= latestVersion; version++ {
+		t.Run("migration-v"+strconv.Itoa(version), func(t *testing.T) {
```

Comment:
One test should be about starting from the current storage node state, not from zero.